src/Modal/index.jsx

```diff
@@ -100,7 +100,19 @@ class Modal extends React.Component {
 
   render() {
     return (
-      <div className={classNames(styles.modal, styles.show, { [styles['modal-open']]: this.state.open })} role="dialog" aria-modal aria-labelledby={this.headerId}>
+      <div
```

Comment:
Nit: you could also consider pulling out `open` from `this.state` like
```
const { open } = this.state;
```
so you don't have to rewrite `this.state.open` everywhere, but again, it doesn't really matter.